README.md

```diff
@@ -82,22 +82,22 @@ await mongoSchedule.disconnect();
-| withHandler      | `function`                       | true      |               | The function to execute.                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| setter           | parameters                                   | mandatory | default value | description                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| ---------------- | -------------------------------------------- | --------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| withName         | `string`                                     | true      |               | The name of the job. Used as a unique identifier.                                                                                                                                                                                                                                                                                                                                                                                                            |
+| withSchedule     | `string`,`number` or `string` (default: `0`) | false     |               | Either this setter or `withCronSchedule()` must be called. Specifies the time interval at which the job is started. Time intervals in human-readable formats (like '1 minute', 'ten days' or 'twenty-one days and 2 hours' ) are accepted. Check documentation of [ human-interval ](https://www.npmjs.com/package/human-interval) library for details. If the job never ran before, the job will run after `firstRunAfter` milliseconds for the first time. |
```

Comment:
We should also add an example where we actually set the `firstRunAfter`. Once in a human readable format, once with just a number.